packages/formspree-react/src/context.tsx

```diff
@@ -80,7 +76,7 @@ export const FormspreeProvider = (props: FormspreeProviderProps) => {
     if (stateStripePromise) {
       setClient(handleCreateClient(stateStripePromise, props.project));
     }
-  }, [stateStripePromise]);
+  }, [props.project, stateStripePromise]);
```

Comment:
Add missing `useEffect` dependency.

This fixes the issue that the client is not updated when the project changes and uses the stale project.